github/issues_comments.go

```diff
@@ -10,6 +10,9 @@ import (
 	"time"
 )
 
+// maxCommentLen is the maximum length of a comment that github will accept without a validation error.
+const maxCommentLen = 65536
```

Comment:
Nitpick, but can we rename this to `maxCommentLength` or `maxCommentSize`? 

Having a trimmed word ("len") next to full words like "comment" looks weird.

See https://github.com/golang/go/blob/master/src/net/http/client.go#L549 for example from Go source code.